www-apps/radicale/radicale-2.1.11-r2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python3_6 python3_7 )
+
+inherit distutils-r1 eutils user systemd
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="https://radicale.org/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+bcrypt"
+
+RDEPEND="sys-apps/util-linux
+	dev-python/vobject[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	bcrypt? ( dev-python/passlib[bcrypt,${PYTHON_USEDEP}] )"
+
+S=${WORKDIR}/${MY_P}
+
+RDIR=/var/lib/${PN}
+
+pkg_pretend() {
+	if [[ -f ${RDIR}/.props && ${MERGE_TYPE} != buildonly ]]; then
+		eerror "It looks like you have a version 1 database in ${RDIR}."
+		eerror "You must convert this database to version 2 format before upgrading."
+		eerror "You may want to back up the old database before migrating."
+		eerror
+		eerror "If you have kept the Gentoo-default database configuration, this will work:"
+		eerror "1. Stop any running instance of Radicale."
+		eerror "2. Run \`radicale --export-storage ~/radicale-exported\`."
+		eerror "3. Run \`chown -R radicale: ~/radicale-exported\`"
+		eerror "4. Run \`mv \"${RDIR}\" \"${RDIR}.old\"\`."
+		eerror "5. Install Radicale version 2."
+		eerror "6. Run \`mv ~/radicale-exported \"${RDIR}/collections\"\`."
+		eerror
+		eerror "For more details, or if you are have a more complex configuration,"
+		eerror "please see the migration guide: https://radicale.org/1to2/"
+		eerror "If you do a custom migration, please ensure the database is cleaned out of"
+		eerror "${RDIR}, including the hidden .props file."
+		die
+	fi
+}
+
+pkg_setup() {
+	enewgroup ${PN}
```

Comment:
remove this and create `acct-user` and `acct-group`: https://wiki.gentoo.org/wiki/Categories_acct-group_and_acct-user

Reply:
So am I supposed to (1) create acct-user/acct-group packages with myself as maintainer, even though I have nothing to do with the user of those accounts; or (2) create them with maksbotan as their maintainer, thus forcing packages into his maintainership unilaterally? I didn’t know the answer to this question so I didn’t migrate at all, for now, since [this devmanual page](https://devmanual.gentoo.org/ebuild-writing/users-and-groups/index.html) said “The historical way … must not be used for new packages” and this is not a new package, only a revbump. But I can create them if you tell me which maintainer I should put on them.

Reply:
Left this alone per your comment below.